src/test/java/com/redhat/lightblue/hook/audit/AbstractMongoTest.java

```diff
@@ -0,0 +1,55 @@
+package com.redhat.lightblue.hook.audit;
+
+import com.mongodb.BasicDBObject;
+import com.mongodb.DB;
+import com.mongodb.DBCollection;
+import com.mongodb.DBObject;
+import com.redhat.lightblue.mongo.test.EmbeddedMongo;
+import com.redhat.lightblue.util.test.AbstractJsonSchemaTest;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+
+
+/**
+ * Created by lcestari on 1/9/15.
+ */
+public abstract class AbstractMongoTest extends AbstractJsonSchemaTest {
```

Comment:
I don't think adding this class is the right solution but will let it go until we remove these abstract mongo tests and move to the in memory mongo resource.